.github/workflows/dapr-test-sdk.yml

```diff
@@ -84,7 +84,11 @@ jobs:
 
             Commit ref: ${{ env.CHECKOUT_REF }}
 
-      - uses: actions/checkout@v3
+      - name: Check out code
```

Comment:
The SDK tests were failing because it was cloning master rather than the PR. This should fix it.